docs/conf/portland/2019/news/schedule-tickets.rst

```diff
@@ -0,0 +1,58 @@
+:template: {{year}}/generic.html
+
+.. post:: April 8, 2019
+   :tags: 2019, portland-2019, schedule, tickets
+
+Announcing Full Schedule, Live Streaming, and Tickets Sold Out
+==============================================================
+
+Hello everyone!
+We're about a month out from the Portland conference and we hope you're getting ready!
+We're writing today to announce the full presentation schedule and to give you the latest registration update.
+
+In the meantime, if it's your first Write the Docs, now's a great time to check out `the Welcome Wagon page <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_, which is chockfull of info to help you get oriented.
+And whether it's your first or fifth conference, it's never too early to check out the `Explore Portland
+<https://www.writethedocs.org/conf/portland/2019/visiting/>`_ page for inspiration on how to make the most of your visit.
+
+Tickets are sold out
+--------------------
+
+Every year we see more enthusiasm from the community, and we are happy and sad at the same time to announce that we are now **sold out** at capacity.
```

Comment:
```suggestion
Every year we see more enthusiasm from the community, and we are happy and sad at the same time to announce that we are now **sold out** at capacity.
```
We're not really sad. Can we figure out another way to say this?

Reply:
Sorry to disappoint those who don't have tickets yet maybe?

Reply:
made a suggestion?